lookout/style/format/rules.py

```diff
@@ -387,3 +517,16 @@ def confidence(v, not_v):
             if new_verbs:
                 new_rules.append(Rule(tuple(new_verbs), stats))
         return new_rules
+
+    def __setstate__(self, state):
+        if "_base_model" not in state:
+            state["_base_model"] = None
```

Comment:
It is never in the state according to `__getstate__` so always `self._base_model = None`